### PR TITLE
add parameter description for parameters in one line/GRASS GIS parameter

### PR DESCRIPTION
**Pull request by anikaweinmann**
With this changes we get an description for each varibable (even though these are in one GRASS GIS parameter) in the actinia templates.
E.g. for the pc-template:
`{
	"id": "default_value",
	"description": "test default value in actinia-gdi",
	"template": {
		"version": "1",
		"list": [
			{
				"module": "r.mapcalc",
				"id": "r.mapcalc_test",
				"inputs": [
					{
						"param": "expression",
						"comment": "output = r.mapcalc result, string; value = raster value (default=0.428), float",
						"value": "{{ output }} = {{ value|default(0.428) }}"
					}
			  ]
			}
		]
	}
}
`
we get:
`{
categories: [
"actinia-module"
],
description: "test default value in actinia-gdi",
id: "default_value",
parameters: {
output: {
description: ". Expression to evaluate [generated from r.mapcalc_expression] - output = r.mapcalc result; value = raster value (default=0.428) [generated from r.mapcalc_expression]",
required: false,
schema: {
type: "string"
}
},
value: {
description: ". Expression to evaluate [generated from r.mapcalc_expression] - output = r.mapcalc result; value = raster value (default=0.428) [generated from r.mapcalc_expression]",
required: false,
schema: {
type: "string"
}
}
},
returns: { }
}`

### CODE REVIEW

File: actinia_gdi/core/gmodulesParser.py [hunk 1]
```diff
@@ -198,7 +198,14 @@ def ParseInterfaceDescription(xml_string, keys=None):
         if keys:
             # case for actinia modules
             key = setVirtualParameterKey(module_id, parameter)
+            key_exists = False
             if key not in keys:
+                for actiniakey in keys:
+                    if actiniakey.startswith(key):
```
Comment (mmacata) [inline actinia_gdi/core/gmodulesParser.py:204]: why is this needed?

Comment (anikaweinmann) [reply actinia_gdi/core/gmodulesParser.py:204]: Cause I added the name to the key. So it is now e.g. actriniakey = r.mapcalc_expression_output and r.mapcalc_expression_value and the key is only r.mapcalc_expression

Comment (mmacata) [reply actinia_gdi/core/gmodulesParser.py:204]: Ok, fine for me

**Comment by mmacata** [comment]
Please merge
